virttest/qemu_devices/qcontainer.py

```diff
@@ -1438,6 +1438,15 @@ def machine_other(cmd=False):
                         cmd += ",accel=%s" % machine_accel
                     if machine_type_extra_params:
                         cmd += ",%s" % machine_type_extra_params.strip(',')
+                    machine_help = process.run("%s -M %s,\?" %
+                                               (self.__qemu_binary,
+                                                machine_type),
+                                               shell=True).stdout_text
+                    opt = "memory-backend"
+                    backend_id = "machine_backend"
```

Comment:
The id is fixed, how do you think about move it to base.cfg, make it clear for user to set params for the memory object ?

Reply:
Make sense. Added it to base.cfg, pls help review again.

Reply:
@yanan-fu what is the use case for user specified id?

Reply:
The id is expected to bz fixed, and distinguish with other memory object. Do you have any other suggestion ?